isisd/isis_route.c

```diff
@@ -602,10 +684,35 @@ static void _isis_route_verify_table(struct isis_area *area,
 	}
 }
 
+static void _isis_route_verify_merge(struct isis_area *area,
+				     struct route_table **tables,
+				     struct route_table **tables_backup,
+				     int tree);
+
 void isis_route_verify_table(struct isis_area *area, struct route_table *table,
-			     struct route_table *table_backup)
+			     struct route_table *table_backup, int tree)
 {
-	_isis_route_verify_table(area, table, table_backup, NULL);
+	int tables_next = 1;
+	int level = area->is_type == IS_LEVEL_1 ? ISIS_LEVEL1 : ISIS_LEVEL2;
+	struct listnode *node;
+	struct flex_algo *fa;
+	struct route_table *tables[255] = {table};
+	struct route_table *tables_backup[255] = {table_backup};
+
+	for (ALL_LIST_ELEMENTS_RO(area->flex_algos->flex_algos, node, fa)) {
+		struct isis_flex_algo_data *data = fa->data;
+		tables[tables_next] =
+			data->spftree[tree][level - 1]->route_table;
+		tables_backup[tables_next] = srcdest_table_init(); // TODO(slankdev)
```

Comment:
What is needed here ?

Reply:
This was prepared by dummying the backup table.

My original implementation does not support Fast-Re-Route for Flex-Algo Algorithm. I added the dummy table because I had to create a dummy backup table at the time of development, otherwise I had crashes.

Since I don't understand the Fast-Re-Route implementation well yet, I didn't have a strong intention to do this either, and I thought I needed to delete it before merge. (or implement Fast-Re-Route for Flex-Algo).